packs/alien-core-bestiary/Hellknight_Degradant_iuovM4PRyAX0NwvI.json

```
@@ -716,7 +716,20 @@
           "license": "OGL",
           "remaster": false
         },
-        "rules": [],
+        "rules": [
+          {
+            "key": "Note",
+            "outcome": [
+              "success",
+              "criticalSuccess"
+            ],
+            "text": "SF2E.NPCAbility.HellknightDegradant.GravitonGlaive.Note",
+            "selector": [
+              "graviton-glaive-attack"
+            ],
+            "title": "SF2E.NPCAbility.HellknightDegradant.GravitonGlaive.Label"
+          }
```

```suggestion
            "text": "{item|description}",
            "selector": [
              "graviton-glaive-attack"
            ],
            "title": "{item|name}"
          }
```
Not worth localizing to cut off the tiny bit of text there.